Common subdirectories: mod_watch-4.3/Contrib and mod_watch-4.3_apache22_mod/Contrib
Common subdirectories: mod_watch-4.3/Img and mod_watch-4.3_apache22_mod/Img
diff -u mod_watch-4.3/Makefile.dso mod_watch-4.3_apache22_mod/Makefile.dso
--- mod_watch-4.3/Makefile.dso	2003-03-14 10:50:15.000000000 +0100
+++ mod_watch-4.3_apache22_mod/Makefile.dso	2007-04-19 13:49:22.000000000 +0200
@@ -13,7 +13,8 @@
 #
 # The location of apxs utility.
 #
-APXS=/home/apache2/bin/apxs
+#APXS=/home/apache2/bin/apxs
+APXS=/usr/local/sbin/apxs
 
 #
 # The location of apachectl utility to stop/start/restart targets.
diff -u mod_watch-4.3/mod_watch.c mod_watch-4.3_apache22_mod/mod_watch.c
--- mod_watch-4.3/mod_watch.c	2003-03-13 22:44:27.000000000 +0100
+++ mod_watch-4.3_apache22_mod/mod_watch.c	2007-11-25 16:12:05.785948411 +0100
@@ -258,7 +258,7 @@
 {
 	char *username;
 
-	if (apr_get_username(&username, uid, p) == APR_SUCCESS)
+	if (apr_uid_name_get(&username, uid, p) == APR_SUCCESS)
 		return username;
 
 	return "unknown-user";
@@ -278,7 +278,7 @@
 {
 	char *groupname;
 
-	if (apr_get_groupname(&groupname, gid, p) == APR_SUCCESS)
+	if (apr_gid_name_get(&groupname, gid, p) == APR_SUCCESS)
 		return groupname;
 
 	return "unknown-group";
@@ -355,7 +355,7 @@
 
 	nbytes = sizeof word;
 	rc = fileReadWord(fp, word, &nbytes);
-	if (!APR_STATUS_IS_SUCCESS(rc))
+	if (rc != APR_SUCCESS)
 		return rc;
 
 	return sscanf(word, format, number) == 1 ? APR_SUCCESS : APR_EGENERAL;
@@ -370,7 +370,7 @@
 
 	nbytes = sizeof word;
 	rc = fileReadWord(fp, word, &nbytes);
-	if (!APR_STATUS_IS_SUCCESS(rc))
+	if (rc != APR_SUCCESS )
 		return rc;
 
 	return sscanf(word, "%lf", number) == 1 ? APR_SUCCESS : APR_EGENERAL;
